packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -350,7 +350,7 @@ class RenderParagraph extends RenderBox with ContainerRenderObjectMixin<RenderBo
   List<InlineSpanSemanticsInformation>? _cachedCombinedSemanticsInfos;
 
   /// The text to display.
-  InlineSpan get text => _textPainter.text!;
+  InlineSpan get text => _textPainter.text ?? const TextSpan();
```

Comment:
This is going to need a unit test that verifies that calling the getter does not crash, and returns an empty span, when the `_textPianter.text` is null.